openmdao/core/system.py

```diff
@@ -3433,7 +3433,9 @@ def list_inputs(self,
                     excludes=None,
                     all_procs=False,
                     out_stream=_DEFAULT_OUT_STREAM,
-                    values=None):
+                    values=None,
+                    print_min=False,
+                    print_max=False):
```

Comment:
Need to make these same changes to the `Case` methods list_inputs()/list_outputs(), which should mirror the `System` methods...